.github/workflows/push-workflow.yaml

```diff
@@ -30,3 +30,57 @@ jobs:
 
       - name: List applications
         run: mabl applications list
+  test-mac:
```

Comment:
Can we use a build matrix here, so we don't have to repeat the build 3x?

Reply:
Done. I also pinned Ubuntu to 20.04